sample_files/oca_dependencies.txt

```diff
@@ -0,0 +1,15 @@
+# List the OCA project dependencies, one per line
+# Add a repository url and branch if you need a forked version
+#
+# Examples
+# ========
+#
+# To depend on the standard version of sale-workflow, use:
+# sale-workflow
+#
+# To explicitely give the URL of a fork, and still use the version specified in
+# .travis.yml, use:
+# sale-workflow https://github.com/OCA/sale-workflow
+#
+# To provide both the URL and a branch, use:
+# sale-workflow https://github.com/OCA/sale-workflow branchname
```

Comment:
I don't think so: I have not seen any `.travis.yml` files in the OCA specifying a hash for a dependency branch. I expect most of the time to have only the branch name in the file.